### ISSUE TITLE
Scylla installation failed on Debian 11

### ISSUE THREAD

**Issue by enaydanov** [scylladb/scylla#9498]
APT failed to install Scylla on Debian 11 because of unmet dependencies for scylla-tools-core:

```
$ sudo apt install scylla
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 scylla-tools-core : Depends: python-yaml but it is not installable
E: Unable to correct problems, you have held broken packages.
```

There is no more `python-yaml` package in the repo, only `python3-yaml`

*Installation details*

Scylla version (or git commit hash): 4.4.dev-0.20201208.f802356572
Repo link: http://downloads.scylladb.com.s3.amazonaws.com/deb/unstable/unified/master/2020-12-08T19:40:17Z/scylladb-master/scylla.list
Debian 11 GCE image link: https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/family/debian-11

**Comment by syuu1228** [comment]
My previous comment was wrong, I found out we actually stopped using PyYAML, I think we just need to drop this dependency:
https://github.com/scylladb/scylla-tools-java/issues/279

**Comment by avikivity** [comment]
Fixed by d949202615dba5723ddf3a059687f0ffca056749 (once it promotes)

**Comment by fruch** [comment]
On debian bookworm (12), we also don't have python2 anymore, we'll need to talk on fixing the cqlsh to work with python3
```
root@acc3687b3bc2:/# apt install scylla                                                                                                                                                                      
Reading package lists... Done                                                                                                                                                                                
Building dependency tree... Done                                                                                                                                                                             
Reading state information... Done                                                                                                                                                                            
Some packages could not be installed. This may mean that you have                                                                                                                                            
requested an impossible situation or if you are using the unstable                                                                                                                                           
distribution that some required packages have not yet been created                                                                                                                                           
or been moved out of Incoming.                                                                                                                                                                               
The following information may help to resolve the situation:                                                                                                                                                 
                                                                                                                                                                                                             
The following packages have unmet dependencies:                                                                                                                                                              
 scylla-tools-core : Depends: python (>= 2.7) but it is not installable                                                                                                                                      
                     Depends: python-yaml but it is not installable                                                                                                                                          
E: Unable to correct problems, you have held broken packages. 
```

**Comment by syuu1228** [comment]
> On debian bookworm (12), we also don't have python2 anymore, we'll need to talk on fixing the cqlsh to work with python3

We should move to python3 until Debian 12 becomes stable version (right now it is "testing" version).
Once we sync more recent code of upstream, cqlsh can run on python3.

**Comment by syuu1228** [comment]
I found that why Debian 12 causes "Depends: python (>= 2.7) but it is not installable" error.
It actually still able to install python2, but "python" virtual package is not installable:
```
$ sudo apt install python
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python-is-python3 2to3 python2-minimal python2 dh-python
```
Therefore, apt cannot resolve "Depends: python (>= 2.7)" anymore.

But I think we can solve the error by replacing  "Depends: python (>= 2.7)"  to "Depends: python2".
It still installable:
```
~$ sudo apt install python2
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cryptsetup-run libcbor0 libusb-0.1-4
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  python2-minimal
Suggested packages:
  python2-doc python-tk
The following NEW packages will be installed:
  python2 python2-minimal
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 58.7 kB of archives.
After this operation, 178 kB of additional disk space will be used.
Do you want to continue? [Y/n] 
```

Note that we should resolve this problem soon, since Ubuntu 22.04LTS(jammy) also has this problem (probably they are using almost same package code).

**Comment by syuu1228** [comment]
> On debian bookworm (12), we also don't have python2 anymore, we'll need to talk on fixing the cqlsh to work with python3
> 
> ```
> root@acc3687b3bc2:/# apt install scylla                                                                                                                                                                      
> Reading package lists... Done                                                                                                                                                                                
> Building dependency tree... Done                                                                                                                                                                             
> Reading state information... Done                                                                                                                                                                            
> Some packages could not be installed. This may mean that you have                                                                                                                                            
> requested an impossible situation or if you are using the unstable                                                                                                                                           
> distribution that some required packages have not yet been created                                                                                                                                           
> or been moved out of Incoming.                                                                                                                                                                               
> The following information may help to resolve the situation:                                                                                                                                                 
>                                                                                                                                                                                                              
> The following packages have unmet dependencies:                                                                                                                                                              
>  scylla-tools-core : Depends: python (>= 2.7) but it is not installable                                                                                                                                      
>                      Depends: python-yaml but it is not installable                                                                                                                                          
> E: Unable to correct problems, you have held broken packages. 
> ```

Fixed this issue at https://github.com/scylladb/scylla-tools-java/commit/6249bfbe2ff9f4be5ddc6cd3af6f191a636784eb

**Comment by avikivity** [comment]
Fixed by e6f7ade60cec56a40bd729604af523ace5a6854c.
